tests/test_url_query.py

```diff
@@ -63,7 +63,7 @@ def test_ampersand_as_value():
 
 
 def test_semicolon_as_separator():
-    u = URL("http://127.0.0.1/?a=1;b=2")
+    u = URL("http://127.0.0.1/?a=1;b=2", separator=";")
```

Comment:
This will fail under some environments unless the behavior isn't made predictable. Also, we probably need tests with combinations of `;` and `&`.